config/webpack.config.base.js

```diff
@@ -11,6 +13,10 @@ module.exports = {
     ]
   },
   resolve: {
-    extensions: ['.js', '.json', '.jsx']
+    extensions: ['.js', '.json', '.jsx'],
+    modules: [
+    	path.join(__dirname,'..','node_modules'),	
+    	path.join(__dirname,'..','node_modules/@parity/ui/node_modules')
```

Comment:
We should not have this link. (Don't see the package in the dependencies anyway)